common/c/targets/esp8266/driver/gpio.c

```diff
@@ -0,0 +1,207 @@
+/*
+ *  Copyright (C) 2014 -2016  Espressif System
+ *
+ */
+
```

Comment:
Will not pass `rats`. Either add license header or add this file to excludes in maven